src/query.ts

```diff
@@ -397,6 +397,19 @@ class Query {
    *     If not specified, default values are chosen by Datastore for the
    *     operation. Learn more about strong and eventual consistency
    *     [here](https://cloud.google.com/datastore/docs/articles/balancing-strong-and-eventual-consistency-with-google-cloud-datastore).
+   * @param {object} [options.gaxOptions] Request configuration options, outlined
+   *     here: https://googleapis.github.io/gax-nodejs/global.html#CallOptions.
+   * @param {boolean} [options.wrapNumbers=false]
+   *     Indicates if the numbers should be wrapped in Int wrapper.
+   * @param {object} [options.integerTypeCastOptions] Configurations to
+   *     optionally wrap `integerValue` in Datastore Int object and optionally
+   *     provide an `integerTypeCastFunction` to handle `integerValue` conversion.
+   *     Note: `integerTypeCastingOptions` values will be ingnored
```

Comment:
Note: `integerTypeCastOptions` is ignored when `options.wrapNumbers` is set.

Reply:
Done

Reply:
@crwilcox I added a few tests in the commit [7c366cc](https://github.com/googleapis/nodejs-datastore/pull/516/commits/7c366ccf20f7421eaa8a750302e8165e8354eb8d) to test the "ignore `integerTypeCastOptions` if `options.wrapNumbers` is not set to `true`"